### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jgarciamesa**
## Pull Request Description

# Checklist
- [ ] Option 1
- [x] Option 2

### CODE REVIEW

**Comment by jgarciamesa** [comment]
Let's discuss this further.

**Review by s-vishnoi — COMMENTED** [review]
Are we sure there are no other funding sources?

**Review by LishaRamon — COMMENTED** [review]
Looking good so far!

File: README.md [hunk 1]
```diff
@@ -13,8 +13,3 @@ The dates so far have been:
 
 The INTERSECT project is supported by NSF awards [2017424](https://www.nsf.gov/awardsearch/showAward?AWD_ID=2017424)
 and [2017259](https://www.nsf.gov/awardsearch/showAward?AWD_ID=2017259).
-
-## Comments -- Activity
-
-These are some comments about the github activity.
-More comments here.
```
Comment (LishaRamon) [inline README.md:20]: What else can we add as a comment?

**Pull request merged**